app/helpers/sites_helper.rb

```diff
@@ -23,7 +23,7 @@ def big_launch_days_number(site)
       end,
     )
 
-    "#{big_day_span} #{small_text}"
+    safe_join([big_day_span, small_text], " ")
```

Comment:
Could there be other autocorrected strings that need a similar treatment - how did you know this one needed fixing? We don't seem to have many interpolation changes in the app code. The only one that caught my eye was [this one](https://github.com/alphagov/transition/blob/e1fcd7fcab4e63c916946f1087968e965ba6fbf6/app/views/mappings/_mappings_table.html.erb#L71), but it looks like it doesn't need to be safe.

Reply:
This one was (luckily) a failing test. After this came up I went through the fixes in the helper files to check none of the others.

It's a bit misleading as a safe autocorrect cop when it applies to html_safe Rails strings.

Thanks for the review 👍